http/1.5/modules/ROOT/pages/http-load-static-res-task.adoc

```diff
@@ -1,29 +1,72 @@
-= To Load a Static Resource
+= Load a Static Resource Example - Mule 4
```

Comment:
Some global items to check for:

- Use of 'in' instead of 'on' as with references to pathnames etc.
- Remember to insert articles ('the' and 'a') as appropriate.
- Use 'that' instead of 'which' in most cases.
- Watch your comma placement.
- If a name is not a product name, then it does not require init caps